knowledge-portal/central/README.md

```diff
@@ -38,8 +39,8 @@ include:
 - All commits from `Knowledge Portal` starts with commit message `docs:....`
 - We check this substring `docs:` in the commit message to ensure the same.
 
-    Notes: `Knowledge Portal` creates new commits for every changed file, if multiple files on `Knowledge Portal` are modified and saved at almost exact time, the pipeline on `central` repo only runs for latest commit(sometimes). 
-    -  To resolve, this issue, we check all commits starting from the latest commit until `docs:` pattern breaks. 
+    Notes: `Knowledge Portal` creates new commits for every changed file, if multiple files on `Knowledge Portal` are modified and saved at almost exact time, the pipeline on `central` repo only runs for latest commit(sometimes, latest theory: this only happens when you do a `Force Sync` opeartion under `Storage` on wikijs). 
```

Comment:
~sometimes, latest theory~ can  be reworded as 'best guess: ...'

Reply:
done